### ISSUE TITLE
Release job failed at 1.25.1

### ISSUE THREAD

**Issue by suzuki-shunsuke** [casey/just#1949]
Release jobs failed at 1.25.1.

https://github.com/casey/just/releases/tag/1.25.1

<img width="448" alt="image" src="https://github.com/casey/just/assets/13323303/ccaf4ed5-7cfa-4fb8-87b1-e649d159d6f7">

https://github.com/casey/just/actions/runs/8217664170/job/22473612069

```
error[E0425]: cannot find value `MAP_32BIT` in crate `libc`
  --> /home/runner/.cargo/registry/src/index.crates.io-6f17d22bba15001f/nix-0.11.1/src/sys/mman.rs:53:9
   |
53 |         MAP_32BIT;
   |         ^^^^^^^^^ not found in `libc`

For more information about this error, try `rustc --explain E0425`.
error: could not compile `nix` (lib) due to 1 previous error
warning: build failed, waiting for other jobs to finish...
Error: Process completed with exit code 101.
```

<img width="338" alt="image" src="https://github.com/casey/just/assets/13323303/aa149c34-3024-46c3-91da-f941dcfad514">

**Comment by cinerea0** [comment]
This seems to be in some way a consequence of pinning ctrlc to 3.1.1 (https://github.com/casey/just/commit/9d6a25152840d1d697009bd5e49f514406c95ef2) since that's the one commit that separates v1.25 from v1.25.1.

**Comment by laniakea64** [comment]
> This seems to be in some way a consequence of pinning ctrlc to 3.1.1

Looks like that was done because of https://github.com/casey/just/issues/1944 , which the reporter solved by using `cargo install --locked just`.  However, `--locked` install of `just` version 1.25.0 would've used `ctrlc` version 3.4.2, not 3.1.1 -
https://github.com/casey/just/blob/6228ff542d6dc52996a79ca895c0f2b998db20a0/Cargo.lock#L240-L242

**Comment by casey** [comment]
Okay, take two! I just published [1.25.2](https://github.com/casey/just/releases/tag/1.25.2), which hopefully fixes this!

**Comment by cinerea0** [comment]
I can confirm that the issue is fixed in 1.25.2, at least in Void's CI.
